procrastinate/worker.py

```diff
@@ -34,11 +34,17 @@ def __init__(
             self.logger = logger
 
     async def run(self) -> None:
-        await self.job_store.listen_for_jobs(queues=self.queues)
+        queues = self.queues
+        await self.job_store.listen_for_jobs(queues=queues)
 
-        self.logger.debug(
-            f"Log start one queues {self.queues}",
-            extra={"action": "start_log_on_queue", "queues": str(self.queues)},
+        if queues:
+            queue_names = f"queues {', '.join(queues)}"
+        else:
+            queue_names = "all queues"
+
+        self.logger.info(
+            f"Starting worker on queues {queue_names}",
+            extra={"action": "start_worker", "queues": queues},
```

Comment:
You really want to use `queues` rather than `queue_names` here in the extra's?

Reply:
yes, for me the structured log is supposed to be structured, no ?

Reply:
Ok I see.